src/config/mod.rs

```diff
@@ -35,6 +35,10 @@ pub fn about_cache_duration() -> usize {
     usize_with_default("ABOUT_CACHE_DURATION", request_cache_duration())
 }
 
+pub fn long_cache() -> usize {
```

Comment:
Any specific reason why here we use a duration based env var, while the others are more use case based (long cache vs about cache). Personally I would stick to use case, but there might be good reasons to have it duration based.

Reply:
It was because of what you mentioned the other day regarding having caches based in duration in the future. If you prefer we can stick to use case for now and then do a full migration of everything when we decide a concrete naming scheme for this.